MapboxNavigation/VisualInstruction.swift

```diff
@@ -5,13 +5,13 @@ import CarPlay
 
 extension VisualInstruction {
     
-    var containsLaneIndications: Bool {
+    public var containsLaneIndications: Bool {
```

Comment:
Without documentation comments, these symbols won’t appear in the docset, so they’re effectively private.